content/docs/introduction/faq.md

```diff
@@ -201,7 +201,7 @@ to configure Prometheus for the amount of memory you have available.
 ### My Prometheus 1.x server reports to be in “rushed mode” or that “storage needs throttling”.
 
 Your storage is under heavy load. Read
-[the section about configuring the local storage](/docs/prometheus/1.8/storage/)
```

Comment:
The question is about "My Prometheus 1.x server" so the current link is correct.

Reply:
@roidelapluie , you're right! I've reverted my change in this commit: https://github.com/prometheus/docs/pull/1725/commits/8fcadfbd343728a209d46c976fbd94b36ee1060f. Thanks.